nss-day-cohort-22/company-website-part-deux-deceptive-eels
# Sean safety page

# to test

1. checkout my branch
1. make sure all of the various navigation buttons work
1. profit
